Add token examples of python (#25)
streamnative/pulsar-examples: cloud/pulsar-admin/README.md

# Overview

> The `pulsar-admin` is a CLI tool written by Java language for the Apache Pulsar project.

Reviewer: ```suggestion
The `pulsar-admin` is a CLI tool written in Java language for the Apache Pulsar project.
```

# Prerequisites

- Pulsar broker 2.7.0-742fc5c9b+

> > You can get this tarball from [bintray](https://bintray.com/streamnative/maven/org.apache.pulsar/2.7.0-742fc5c9b). When the 2.6.1 release is complete, you can also use the official 2.6.1 version.

Reviewer: ```suggestion
> You can get this tarball from [binary](https://bintray.com/streamnative/maven/org.apache.pulsar/2.7.0-742fc5c9b). When Pulsar 2.6.1 is released, you can also use the official 2.6.1 version.
```

Author: `bintray` is the name of a website, not typo.

# Usage

> The `pulsar-admin` supports to connect to Pulsar cluster through Token, the example as follows:

Reviewer: ```suggestion
The `pulsar-admin` supports to connect to Pulsar cluster through Token, as shown below:
```

```shell script
./bin/pulsar-admin \
    --url WEB_SERVICE_URL \
    --auth-params AUTH_PARAMS \
    tenants list
```

> How to get the `WEB_SERVICE_URL` and `AUTH_PARAMS` fields, please reference to **how to get Token options**.

Reviewer: ```suggestion
For details about how to get the `WEB_SERVICE_URL` and `AUTH_PARAMS` fields, refer to **how to get Token options**.
```

and use the cross reference about [how to get Token options](). same as below.
